test/unit/math/prim/scal/prob/bernoulli_test.cpp

```diff
@@ -6,37 +6,68 @@
 TEST(ProbDistributionsBernoulli, error_check) {
   boost::random::mt19937 rng;
   EXPECT_NO_THROW(stan::math::bernoulli_rng(0.6,rng));
+  EXPECT_NO_THROW(stan::math::bernoulli_logit_rng(-3.5,rng));
```

Comment:
Revert the changes to `bernoulli_test.cpp` and this should all be in `bernoulli_logit_test.cpp`.